blaze/_version.py

```diff
@@ -0,0 +1,191 @@
+
```

Comment:
This is all generated or copy/pasted from versioneer right? So don't review this module?